icons/poi_p.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="32" xmlns="http://www.w3.org/2000/svg">
+ <path d="M 0,0 V 32 H 6 V 22 h 6 A 12,11 0 0 0 24,11 12,11 0 0 0 12,0 H 4 Z m 6,6 h 7 a 5,5 0 0 1 5,5 5,5 0 0 1 -5,5 H 6 Z" style="fill:#003f87;fill-opacity:1;stroke-width:2"/>
```

Comment:
This path hugs three sides of the viewbox and is relatively thick, causing it to appear much larger than the martini glass icon, even when both are rendered at nominally the same size. For balance, this path should be shrunk down by a few pixels, as should the health cross icon, which has a similarly thick path.